src/emmetHelper.ts

```diff
@@ -521,6 +521,14 @@ export function isAbbreviationValid(syntax: string, abbreviation: string): boole
 	if (syntax === 'jsx') {
 		return (jsxAbbreviationStartRegex.test(abbreviation) && htmlAbbreviationRegex.test(abbreviation));
 	}
+	
+	// Fix for jinja syntax https://github.com/microsoft/vscode/issues/179422
+	if (syntax === 'html') {
+		if (/^({%|{#|{{)/.test(abbreviation)) {
+		  return false;
```

Comment:
The indent seems a bit off here. Also, the isStyleSheet check above already covers CSS and CSS-related languages and returns early, so if it is possible to type jinja syntax in other HTML-related languages too, then the outer `syntax === 'html'` condition can be removed.

Reply:
I don't think it has a meaning at any of the html related languages, that's why that's why I went with checking the syntax to be html, what do you think? @rzhao271

Reply:
The indentation seems to be only off in the diff here, it looks fine locally
<img width="980" alt="image" src="https://github.com/microsoft/vscode-emmet-helper/assets/46695441/69e5e9d4-d73e-4c13-aebc-dc34104cafb2">

Reply:
If the abbreviation shouldn't be parsed as an Emmet abbreviation in any HTML-related language, then the outer condition can be removed.

Reply:
resolved